### ISSUE TITLE
Can't view images in documentation

### ISSUE THREAD

**Issue by NebuPookins** [grapl-security/grapl#147]
I'm trying to read about the project at https://insanitybit.github.io/2019/03/09/grapl but I'm unable to see many of the images, which makes it hard to follow along with the article.

For example, the article links to https://d2mxuefqeaa7sj.cloudfront.net/s_A34F006A82484ABC31C78D15F90D61A26E825A6B4715DF2C484A9992F8A03A0F_1551942907936_Screenshot+from+2019-03-06+23-14-54.png but when I try to load that file I get

```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>7C585F52B5EBAA51</RequestId>
<HostId>6T1+U4XDa5s76WC9+9qfDfKoc8iO21VRppu/N+gO5tgVKq7KGT+pNL9efE65yGJjwf5QgAA8/OU=</HostId>
</Error>
```

**Comment by insanitybit** [comment]
Hey, thanks for the report. Looks like some of the images have been moved since that post, which is from a while back.

I'll get to fixing that, but in the meantime, you may find posts on graplsecurity.com/blog , and that particular post was based on a conference talk I've given a few times, most recently at BSidesSF.

https://www.youtube.com/watch?v=uErWRAJ4I4w

**Comment by wimax-grapl** [comment]
I'm getting our docs back in working order, will try and fix this soon!

**Comment by colin-grapl** [comment]
@wimax-grapl Any updates?

**Comment by NebuPookins** [comment]
I checked the link https://insanitybit.github.io/2019/03/09/grapl again and I'm able to see some, but not all the images.

This image loads for me: https://i2.wp.com/sourcedexter.com/wp-content/uploads/2017/04/TensorFlow-graph1.jpg?ssl=1

But this image does not load: https://d2mxuefqeaa7sj.cloudfront.net/s_A34F006A82484ABC31C78D15F90D61A26E825A6B4715DF2C484A9992F8A03A0F_1552159733616_Screenshot+from+2019-03-09+11-26-28.png

And the error with the 2nd image is:

```xml
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>JH69DYKHFXKF22E9</RequestId>
<HostId>HfKySPtdUTYTnsi2mNJnleV7S8m6R0ph15+LO05TubjeB6URWioNluJiQAOCtpfn0cVFLsa795E=</HostId>
</Error>
```

**Comment by insanitybit** [comment]
Ah, well that's my personal blog, which I don't do a great job of maintaining. I would suggest the youtube videos of the conference talk, which covers all of that same content.

https://www.youtube.com/watch?v=uErWRAJ4I4w
